doc/Glossary/index.md

```diff
@@ -5,74 +5,236 @@ permalink: /:path/
-| Transport channel                                       | A Kaa client component responsible for communication with a Kaa server. Refer to Transports for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
```

Comment:
A virtual representation of an Endpoint, that is part of the [actor gateway]({{root_url}}Glossary/#actor-gateway) architecture.
An actor typically communicates with the actual device and represents it to [Kaa server]({{root_url}}Glossary/#kaa-server).